ccswbs/hjckrrh
# Iss156 Book Validation Patch

Created a patch for the book module and saved it in `profiles/ug/patches/book_validation.patch`. The patch has also been added locally to `drupal-org-core.make` so it will apply when `drush make profile/ug/drupal-org-core.make` is run. The patched book module as been committed to the repo.

## Test Procedure
1. Checkout branch `iss156-patch`
2. Enable the `book` module
3. Create a book page, making it into a new book
4. Create a second book page and make it a child of the first page
5. Ensure an "Up" link is visible on the child page and that it takes you to the parent
6. Ensure the child page validates

## Automated Test
The automated test is in ccswbs/testing#19. Note that the `ug_testing_api` and `book` modules will need to be enabled for the test to run properly.
